solidity.html.markdown

```diff
@@ -786,11 +786,18 @@ sha256("def");
 
 // 12. LOW LEVEL FUNCTIONS
 // call - low level, not often used, does not provide type safety
-successBoolean = someContractAddress.call('function_name', 'arg1', 'arg2');
+uint u = 123;
+address a = this;
+successBoolean = someContractAddress.call(bytes4(sha3('function_name(uint256,address)')), u, a);
```

Comment:
It'd be nice if you could split out the `bytes4(sha3` call somewhere and explain it.